code/controllers/subsystem/persistent_paintings.dm

```diff
@@ -113,6 +113,11 @@ SUBSYSTEM_DEF(persistent_paintings)
 	/// A list of /datum/paintings saved or ready to be saved this round.
 	var/list/paintings = list()
 
+	/// A list of paintings' data for paintings that are currently stored in the library.
+	var/list/painting_data = list()
+	/// A list of paintings' data for paintings that are currently stored in the library, with admin metadata
+	var/list/admin_painting_data = list()
```

Comment:
what happens when a new painting is added?

Reply:
Addressed in new commit (also made me realize that `painting_data` is used as a local variable elsewhere in the file)